bot/main.py

```diff
@@ -210,25 +244,42 @@ async def backend_info(ctx):
     )
 
 
-def is_academic(ctx: commands.Context):
-    """Checks if the server is an academic server."""
+@backend_info.error
+async def backend_info_error(ctx, error):
+    """If the author of the message is not a bot admin then reply accordingly."""
+    if (
+        isinstance(error, commands.CheckFailure)
+        and isinstance(error.original, CheckFailedException)
+        and error.original.check_name == "is_author_bot_admin"
+    ):
+        author_id = str(ctx.message.author.id)
+        await ctx.reply(f"{author_id} is not a bot admin.")
+    else:
+        await ctx.reply("Some check failed.")
+
+
+def is_academic_or_bot_admin(ctx: commands.Context):
+    """Checks if the server is an academic server or if the author is a bot admin."""
     server_config = get_config(str(ctx.guild.id))
+    user_id = ctx.message.author.id
 
     if server_config is None:
-        return False
+        if not is_bot_admin(str(user_id)):
+            raise CheckFailedException("is_academic_or_bot_admin")
     return server_config.get("is_academic", False)
```

Comment:
The `is_bot_admin` block will not run in this codepath (when server_config is not None), it would be needed here as well

Reply:
Ah yes, I'll fix this.